### DISCUSSION TITLE
redirect from sub-path internationalization to sub domain

### DISCUSSION THREAD

**Discussion by nareshopoyi** [vercel/next.js#37707] [Help]
recently we changed from sub-path to sub domain ex.
**https://opoyilocal.com:3000/hindi/topic/newtopic/** to **https://hindi.opoyilocal.com:3000/topic/newtopic/**

i still can access the old pages **https://opoyilocal.com:3000/hindi/topic/newtopic/**. we want to redirect this to **https://hindi.opoyilocal.com:3000/topic/newtopic/**

**Comment by icyJoseph** [comment]
You mean something like:

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  trailingSlash: true,
  async redirects() {
    return [
      {
        source: "/hindi/:path*/",
        destination: "https://hindi.example.com/:path*/",
        permanent: false, // I think you may want `true` here
      },
    ];
  },
};

module.exports = nextConfig;
```

I think that should work.

**Reply by nareshopoyi** [reply]
Thanks Joseph, but its not working

**Reply by icyJoseph** [reply]
ah, perhaps I missed the trailing slashes? And of course, assuming you are deploying this to the old service as well, the one that uses the sub-path.
